other tasks/task9.js

```diff
@@ -0,0 +1,29 @@
+(() => {
+  'use strict'
+
+  let uniqueName;
+
+  kintone.events.on(['app.record.create.submit', 'app.record.edit.submit'], (event) => {
+    //"uniqueName" = 現在のレコードの重複禁止項目の値
+    uniqueName = event.record.重複禁止項目.value;
+    console.log(uniqueName);
+
+    //"uniqueName"と重複する重複禁止項目値を持ったレコードだけを抽出
+    const params = {
+      'fields': '重複禁止項目',
+      'app': 31,
```

Comment:
```suggestion
      app: kintone.app.getId(),
```

同一アプリなので、`getId()` でIDを指定しましょう！